# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1392625563 18000
#      Mon Feb 17 03:26:03 2014 -0500
# Branch stable
# Node ID 2fafb80f3616228df9ffac54b154ccdd826a4e63
# Parent  85d31344fb23725791bc687337295d47416b9c17
Added tag rc-3-8-1-2 for changeset 85d31344fb23

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -78,8 +78,9 @@ cc5a7d1233f3acea85648baeb754fc0e8f225225
 b29b10fbb7448cdfe29322446e1a589e7fe1a40a release-3-6-4
 4e50bd2946d8563d3e201cc04b3ba0720c991b06 ss-3-7-4
 608e307b49149b32a6d09c2f06493d04d3af9be4 ss-3-7-5
 3a9efb68272df556dccb84c87933dd8238e88902 ss-3-7-6
 cc13924a4266fb0359f59fabdce11071e6051d48 ss-3-7-7
 d734216aa2b1022c190e6b07d7ccdd59fe15678f rc-3-8-0-1
 f1fa148766d623f7f24cf541ea8b88d8c1b63d33 rc-3-8-0-2
 b72bcf5f78cc2231e9bf2735d3b8d92779fc4dff release-3-8-0
+85d31344fb23725791bc687337295d47416b9c17 rc-3-8-1-2
